pandas/core/arrays/categorical.py

```diff
@@ -1607,6 +1607,9 @@ def fillna(self, value=None, method=None, limit=None):
         -------
         filled : Categorical with NA/NaN filled
         """
+        value, method = validate_fillna_kwargs(
```

Comment:
Note, I added this keyword since it's *possible* have a tuple / list for a `category`.